root/meta/naming/CMakeLists.txt

```diff
@@ -1 +1,12 @@
-ROOTTEST_ADD_AUTOMACROS(DEPENDS namingMatches.cxx)
+ROOTTEST_ADD_AUTOMACROS(DEPENDS namingMatches.cxx
+                        EXCLUDE execCheckNaming)
+
+if(MSVC)
+    set(EXECCHECKNAMING_OUTREF execCheckNaming_win32.ref)
```

Comment:
If I remember correctly we can auto detect the sizeof(long) and adjust the name accordingly?  (See the use of ROOTBITS in cmake/modules/RoottestMacros.cmake; eg for example is _win32 if th file exist and the platform is windows).

Reply:
It's not only a 32 bit issue, in this case the symbols (mangled names) are also different between Windows and Linux

Reply:
I'll check how to test if a file exists in CMake. IIUC you suggest to test if it is Windows, if it is 32 bit, and if the ref file exists?

Reply:
Sure (i.e. the same checks you are doing now, just in one single place).

Reply:
Done

Reply:
I am a bit confused.   On my view of the page I still see a lot of conditional setting of reference file names .. what am I missing?

Reply:
They are outdated. See https://github.com/root-project/roottest/pull/554/commits/b3a16c5c8e7b1d9bcd8f50a2aa195759c886fe68

Reply:
I.e. there is a small yellow `Outdated` box after the file name `root/meta/naming/CMakeLists.txt` `Outdated`